test/evaluators/AggregateEvaluator.test.ts

```diff
@@ -33,6 +33,10 @@ function makeAggregate(aggregator: string, distinct = false, separator?: string)
   };
 }
 
+function nonLiteral(): RDF.Term {
+  return namedNode('http://www.w3.org/2001/XMLSchema#integer');
```

Comment:
This is confusing :P I would just use something like `http://example.org/` instead.